spring-boot/admin/src/main/resources/templates/admin/fragments/proxies/list.html

```diff
@@ -143,7 +143,7 @@ <h2 th:text="#{admin.proxies.list.title}">[Proxies Administration]</h2>
                                     <td class="text-center "><a th:href="|@{/admin/proxies/}${proxy.id}|"
                                                                 class="fa fa-bar-chart"/>
                                     </td>
-                                    <td th:text="${proxy.timeOfLastSuccessfulCallToAdmin != null ? #dates.format(new java.util.Date(proxy?.timeOfLastSuccessfulCallToAdmin)) : #{admin.dashboard.gatewayhealthmonitor.notconnectedgateway}}"></td>
+                                    <td th:text="${proxy.timeOfLastSuccessfulCallToAdmin != null ? #dates.format(new java.util.Date(proxy?.timeOfLastSuccessfulCallToAdmin)) : #messages.msg('admin.dashboard.gatewayhealthmonitor.notconnectedgateway.text')}"></td>
```

Comment:
This lines is not readable, too many instruction in one line which make difficult to read

Reply:
Reminder: Use newlines

Reply:
I added newlines to improve readability, please check it @perezdf